### PR TITLE
Add a deploy button.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Comment by almet** [comment]
Hi! As I was saying this morning during the standup, I'm not really in favor of having a link to heroku on our landing page.

However, having a "deploy" button that links to a section of the documentation where different ways to deploy are listed seems a more acceptable idea to me.

I can do these changes if you don't want to make them.

**Comment by Natim** [comment]
@almet is it closer to what you had in mind?

**Comment by Natim** [comment]
deploy

deploy

**Comment by almet** [comment]
We need to chose between "get started" and "deploy now", otherwise the text is too long.

**Comment by Natim** [comment]
Let's do some A/B testing then :+1:

File: docs/tutorials/run-kinto.rst [hunk 1]
```diff
@@ -54,7 +66,7 @@ with a *PostgreSQL* container.
 
 ::
 
-    wget https://raw.githubusercontent.com/Kinto/kinto/master/docker-compose.yml
+    wget http://kinto-storage.org/docker-compose.yml
```
Comment (leplatrem) [inline docs/tutorials/run-kinto.rst]: How is this file published ? Updated ?

File: docs/tutorials/run-kinto.rst [hunk 2]
```diff
@@ -95,6 +107,7 @@ Then install the package using the default configuration:
 
     pip install kinto
     kinto init
+    kinto migrate
```
Comment (leplatrem) [inline docs/tutorials/run-kinto.rst]: Good catch!

**Comment by leplatrem** [comment]
I also vote for two buttons, even if they point to the same page :) but my opinion may not represent most of our users

File: docs/tutorials/run-kinto.rst (hunk 1 above)
Comment (leplatrem) [inline docs/tutorials/run-kinto.rst]: From what I understand we can leave the «official» docker compose

Comment (leplatrem) [inline docs/tutorials/run-kinto.rst]: (once #376 is merged)

**Comment by leplatrem** [comment]
We could also just leave «Get started», and then the «Deploy on heroku» paragraph becomes very obvious on the first look

File: docs/tutorials/run-kinto.rst (hunk 1 above)
Comment (Natim) [inline docs/tutorials/run-kinto.rst]: The file is just committed in the kinto-website repository and gh-pages branch.

**Pull request merged**